contracts/MintableToken.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts v4.4.1 (token/ERC20/ERC20.sol)
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/presets/ERC20PresetMinterPauser.sol";
+
+contract MintableToken is ERC20PresetMinterPauser{
+    constructor (string memory name, string memory symbol) ERC20PresetMinterPauser (name, symbol){}
```

Comment:
Best to hardcode name and symbol in contract itself so that people looking at contract code know what token it is